File: core/src/com/symbol/ecs/component/map/ToggleTileComponent.java
package com.symbol.ecs.component.map;

import com.badlogic.ashley.core.Component;
import com.badlogic.gdx.utils.Pool;

public class ToggleTileComponent implements Component, Pool.Poolable {

    public int id = 0;
    public boolean toggle = true;

    @Override
    public void reset() {
        id = 0;
        toggle = true;
    }

}

File: core/src/com/symbol/ecs/component/map/ToggleTileComponent.kt
package com.symbol.ecs.component.map

import com.badlogic.ashley.core.Component
import com.badlogic.gdx.utils.Pool

class ToggleTileComponent : Component, Pool.Poolable {

    var id: Int = 0
    var toggle: Boolean = true

    override fun reset() {
        id = 0
        toggle = true
    }

}
